usaspending_api/common/helpers/spark_helpers.py

```diff
@@ -401,6 +414,19 @@ def get_jdbc_url():
     return get_jdbc_url_from_pg_uri(CONFIG.DATABASE_URL)
 
 
+def get_broker_jdbc_url(databroker_uri: str)->str:
+    """Getting a JDBC-compliant Broker Postgres DB connection string hard-wired to the POSTGRES vars set in CONFIG"""
+    if not CONFIG.DATA_BROKER_DATABASE_URL:
+        raise ValueError("DATABASE_URL config val must provided")
```

Comment:
`DATA_BROKER_DATABASE_URL` config val must be provided

Reply:
Updated, thank you!